ledger/api-server-damlonx/reference-v2/BUILD.bazel

```diff
@@ -137,6 +137,7 @@ server_conformance_test(
         "--exclude=LotsOfPartiesIT",
         "--exclude=TimeIT",
         "--exclude=ConfigManagementServiceIT",
+        "--exclude=CommandSubmissionCompletionIT",
```

Comment:
What's the reason for not running the test for the reference server?

Reply:
Addressed by https://github.com/digital-asset/daml/pull/3901/commits/75ab721d93ec36ae708c9c717ccda88c9a728164

The test does not represent a behavior that Ledger API implementations have to assume.

Furthermore, the behavior of checkpoints will soon be changed by the new time model.

We therefore decided to drop this test.